src/compiler-rt

```diff
@@ -1 +1 @@
-Subproject commit 57315f7e07d09b6f0341ebbcd50dded6c20d782f
```

Comment:
Yes please, Manishearth. If you could rebase it for me that would be great. I'm away with terrible internet and no dev machine at the moment.